app/bundles/EmailBundle/Swiftmailer/Momentum/DTO/TransmissionDTO/RecipientDTO.php

```diff
@@ -122,9 +122,16 @@ public function jsonSerialize()
         if (count($this->metadata) !== 0) {
             $json['metadata'] = $this->metadata;
         }
-        if (count($this->substitutionData) !== 0) {
+
+        if (count($this->substitutionData) === 0) {
+            // `substitution_data` is required but Sparkpost will return the following error with empty arrays:
+            // field 'substitution_data' is of type 'json_array', but needs to be of type 'json_object'
+            $json['substitution_data'] = new \stdClass();
+
```

Comment:
```suggestion
```
Travis is complaining about this empty row. Remove it, please.

Reply:
@alanhartless